### PR TITLE
Increased max retry police to s3

### PR DESCRIPTION
**Pull request by fragosoluana**
Trying to increase the max retry to make s3 requests more stable.

Related PR: https://github.com/Yelp/nrtsearch/pull/502

### CODE REVIEW

File: src/main/java/com/yelp/nrtsearch/tools/nrt_utils/state/StateCommandUtils.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ public static AmazonS3 createS3Client(
         new RetryPolicy(
             PredefinedRetryPolicies.DEFAULT_RETRY_CONDITION,
             PredefinedRetryPolicies.DEFAULT_BACKOFF_STRATEGY,
-            5,
```
Comment (sarthakn7) [inline src/main/java/com/yelp/nrtsearch/tools/nrt_utils/state/StateCommandUtils.java:98]: Can we get this parameter from a config?

Comment (fragosoluana) [reply src/main/java/com/yelp/nrtsearch/tools/nrt_utils/state/StateCommandUtils.java:98]: Probably. Should we test if this change will actually fix the tron jobs first? If yes, we can create another PR to get this param from a config file.

Comment (sarthakn7) [reply src/main/java/com/yelp/nrtsearch/tools/nrt_utils/state/StateCommandUtils.java:98]: Why not just have one PR which gets the parameter value externally? It's never a good practice to hard-code these values in code.

File: src/main/java/com/yelp/nrtsearch/tools/nrt_utils/incremental/DeleteIncrementalSnapshotsCommand.java [hunk 2]
```diff
@@ -103,6 +103,12 @@ public class DeleteIncrementalSnapshotsCommand implements Callable<Integer> {
       description = "Print file deletions, instead of applying to S3")
   private boolean dryRun;
 
+  @CommandLine.Option(
+      names = {"--maxErrorRetry"},
```
Comment (sarthakn7) [inline src/main/java/com/yelp/nrtsearch/tools/nrt_utils/incremental/DeleteIncrementalSnapshotsCommand.java:107]: Nit: just call these parameters `maxRetry`

**Review by sarthakn7 — APPROVED** [review]

**Review by sarthakn7 — CHANGES_REQUESTED** [review]
Can you please run `./gradlew clean spotlessApply`?

**Review by sarthakn7 — APPROVED** [review]

**Pull request merged**